test-harness/docker-compose.yml

```diff
@@ -20,4 +20,24 @@ services:
     volumes:
       - ".docker/postgresdb/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d"
     environment:
-      POSTGRES_PASSWORD: postgres
\ No newline at end of file
+      POSTGRES_PASSWORD: postgres
+
+  sqlserver:
+    image: conductor:sqlserver
+    build: 
+      context: ../sqlserver-persistence/docker
+      dockerfile: sqlserver/Dockerfile
+    restart: always
```

Comment:
restart defined twice

Reply:
Fixed in https://github.com/TwoUnderscorez/conductor/commit/44bcafaac894648992c2e16e754a252d35d8e79b